aws/resources/lambda_layer.go

```diff
@@ -0,0 +1,143 @@
+package resources
+
+import (
+	"context"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/lambda"
+	"github.com/gruntwork-io/cloud-nuke/config"
+	"github.com/gruntwork-io/cloud-nuke/logging"
+	"github.com/gruntwork-io/cloud-nuke/report"
+	"github.com/gruntwork-io/cloud-nuke/telemetry"
+	commonTelemetry "github.com/gruntwork-io/go-commons/telemetry"
+)
+
+func (ll *LambdaLayers) getAll(c context.Context, configObj config.Config) ([]*string, error) {
+	var layers []*lambda.LayersListItem
+	var names []*string
+
+	err := ll.Client.ListLayersPages(
+		&lambda.ListLayersInput{}, func(page *lambda.ListLayersOutput, lastPage bool) bool {
+			for _, layer := range page.Layers {
+				logging.Logger.Debugf("Found layer! %s", layer)
+
+				if ll.shouldInclude(layer, configObj) {
+					layers = append(layers, layer)
+				}
+			}
+
+			return !lastPage
+		})
+
+	if err != nil {
+		return nil, err
+	}
+
+	for _, layer := range layers {
+		err := ll.Client.ListLayerVersionsPages(
+			&lambda.ListLayerVersionsInput{
+				LayerName: layer.LayerName,
+			}, func(page *lambda.ListLayerVersionsOutput, lastPage bool) bool {
+				for _, version := range page.LayerVersions {
+					logging.Logger.Debugf("Found layer version! %s", version)
+					names = append(names, layer.LayerName)
```

Comment:
Each name in this context should technically be `layer name x version`, since layers can have multiple versions.

Reply:
Can you elaborate what you mean herE?

Reply:
Yup - so currently the output is just the identifier which is the layer's name. There could be potentially multiple rows of the same identifier or layer name since there can be multiple versions of it.

So for a layer `testlayer` with two versions, it would look like this since version isn't displayed: 

```
# Found AWS Resources
┌────────────────────────────────────────┐
| Resource Type | Region    | Identifier |
| -------------------------------------- |
| lambda_layer  | us-west-1 | testlayer  |
| lambda_layer  | us-west-1 | testlayer  |
```

Reply:
Ah I see. Can you add this information as a comment for future readers as well?